### PR TITLE
chore(md): updated readme and contributing guidelines

### PR DESCRIPTION
**Pull request by ovflowd**
This PR introduces changes to the README and the CONTRIBUTION guidelines.

This PR needs a review and approval from @nodejs/website as it changes the Approval Requirements, Commit Guidelines and PR guidelines.

### CODE REVIEW

**Comment by ovflowd** [comment]
cc @nodejs/website

File: CONTENT_VS_CODE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Content vs. Code
+
+The **Website Working Group** is primarily concerned with the code and overall structure of the website.
```
Comment (richardlau) [inline CONTENT_VS_CODE.md]: The Website team isn't a working group. It used to be but was dechartered https://github.com/nodejs/TSC/pull/761.

File: CONTENT_VS_CODE.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+The content of the website comes from a variety of working groups (Evangelism, Core, i18n, etc). The Website WG defers to these WGs on matters of content and routinely adds collaborators from these working groups as they add and improve content on the website. In other words, the Website WG is not an _editorial_ Working Group except when no other Working Group has taken responsibility for a content area.
```
Comment (richardlau) [inline CONTENT_VS_CODE.md]: See https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md#current-working-groups for a list of current working groups -- we don't need to (and should not) duplicate that list here, but "Evangelism" and "Core" are not working groups.

**Review by richardlau — COMMENTED** [review]
I realize these were moved from previous content, but they're outdated and should be updated.

File: README.md [hunk 3]
```diff
@@ -1,49 +1,46 @@
...
+  <br />
+  <br />
+</p>
 
 ## What is this repo?
 
-[nodejs.org](https://nodejs.org/) by the [OpenJS Foundation](https://openjsf.org/) builds on the merged community's past website projects to form a self-publishing, community-managed version of the previous site.
-
-On a technical level, inspiration has been taken from the `iojs.org` repo while design and content has been migrated from the old [nodejs.org repo](https://github.com/nodejs/nodejs.org-archive). These technical changes have helped to facilitate community involvement and empower the foundation's internationalization communities to provide alternative website content in other languages.
+[Nodejs.org](https://nodejs.org/) by the [OpenJS Foundation](https://openjsf.org/) builds on the merged community's past website projects to form a self-publishing, community-managed version of the previous site.
```
Comment (richardlau) [inline README.md]: I'm not sure this sentence is needed. The community split referenced by "merged community" was back in 2014/2015.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,62 +1,181 @@
...
+  > build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+* Commit messages **must** start with a capital letter
+* Commit messages **must not** end with a period `.`
+
+## Pull Request Policy
+
+### Before merging
+
+* Pull Requests must be open for at least 48 hours.
+  * Pull reuqests might be immediately merged if they contain critical bug fixes, short erratas (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
```
Comment (EmirBoyaci) [inline CONTRIBUTING.md]:
```suggestion
  * Pull requests might be immediately merged if they contain critical bug fixes, short erratas (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
```
just a small typo fix :)

Comment (AugustinMauroy) [reply CONTRIBUTING.md]: Oh yes it's from nodejs.dev

File: README.md (hunk 3 above)
Comment (AugustinMauroy) [reply README.md]: mmmm 🤔
Total remove or update?

File: CONTENT_VS_CODE.md (hunk 1 above)
Comment (AugustinMauroy) [reply CONTENT_VS_CODE.md:3]: Oh yess just

File: CONTRIBUTING.md (hunk 4 above)
Comment (ovflowd) [reply CONTRIBUTING.md]: What, Augustin?

Comment (AugustinMauroy) [reply CONTRIBUTING.md]: I just saw that the error is on the nodejs.dev repo. I don't know if you made a copy and paste for this part.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,62 +1,181 @@
...
+
+  ```bash
+  git checkout -b name-of-your-branch
+  ```
+
+6. Run the following to install the dependencies and start a local preview of your work.
+
+  ```bash
+  npm install # installs this project's dependencies
+  npm start # starts a preview of your local changes
```
Comment (mikeesto) [inline CONTRIBUTING.md]:
```suggestion
  npm run serve # starts a preview of your local changes
```
Should this be npm run serve? It seems like the current `npm start` script serves a built version

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,62 +1,181 @@
...
+  ```
+
+7. Perform a merge to sync your current branch with the upstream branch.
+
+ ```bash
+git fetch upstream
+git merge upstream/main
+```
+
+8. Run `npm test` to confirm that tests, linting, and formatting are passing. See [here](#-debugging-failing-checks) to fix failures. Ensure that your tests are passing before making a PR.
```
Comment (mikeesto) [inline CONTRIBUTING.md]: We don't have a test script in this repo atm

**Comment by ovflowd** [comment]
Going to update this doc asap. Just relaxing today after a long trip. Thank you all for the feedback so far :)

BTW, @mikeesto an off-topic thingy, but I sent you a DM on Twitter :)

File: CONTRIBUTING.md (hunk 6 above)
Comment (AugustinMauroy) [reply CONTRIBUTING.md]: I think we can put it in comment and uncomment when we add unit test on this repo. What do you think?

Comment (ovflowd) [reply CONTRIBUTING.md]: Comments aren't part of the CommonMark specification, I will remove the point 8.

File: CONTENT_VS_CODE.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+## Content vs. Code
+
+The **Website Team** (@nodejs/website) is primarily concerned with the code and overall structure of the website.
```
Comment (EmirBoyaci) [inline CONTENT_VS_CODE.md:3]: Based on https://github.com/nodejs/nodejs.org/pull/5101#discussion_r1132339070 I think this should be edited

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,62 +1,172 @@
...
+### Commit Message guidelines
+
+- Commit messages must include a "type"
+  > build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test
+- Commit messages **must** start with a capital letter
+- Commit messages **must not** end with a period `.`
+
+## Pull Request Policy
+
+### Before merging
```
Comment (EmirBoyaci) [inline CONTRIBUTING.md:117]: Maybe also include "Commits should be signed." warning here?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,62 +1,172 @@
...
-become involved in the discussion and review process if they wish.
+## Becoming a collaborator
 
-The default for each contribution is that it is accepted once no collaborator has an objection.
-During review collaborators may also request that a specific contributor who is most versed in a
-particular area gives a "LGTM" before the PR can be merged. There is no additional "sign off"
-process for contributions to land. Once all issues brought by collaborators are addressed it can
-be landed by any collaborator.
+- Collaborators must be actively contributing to the project
+- A Pull Request must be opened on the @nodejs/nodejs.org README file adding the new collaborator to the list (note the order of names)
```
Comment (EmirBoyaci) [inline CONTRIBUTING.md:142]: AFAIS collaborator section is not included in current README. See: https://github.com/nodejs/nodejs.org/blob/main/README.md

File: CONTENT_VS_CODE.md (hunk 7 above)
Comment (ovflowd) [reply CONTENT_VS_CODE.md:3]: This got edited already, and it's still true. We are primarily focused on the structure of the website, not the content.

File: CONTRIBUTING.md (hunk 8 above)
Comment (ovflowd) [reply CONTRIBUTING.md:117]: We added this on "Commit Guidelines" already.

File: CONTRIBUTING.md (hunk 9 above)
Comment (ovflowd) [reply CONTRIBUTING.md:142]: Yup, that's correct. It will in the future :)

**Comment by ovflowd** [comment]
cc @richardlau requesting re-review whenever you get some time 🙇

**Review by mikeesto — APPROVED** [review]

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,62 +1,174 @@
...
+## Becoming a collaborator
 
-For non-trivial contributions, pull requests should sit for at least 36 hours to ensure that
-contributors in other timezones have time to review. Consideration should also be given to
-weekends and other holiday periods to ensure active collaborators all have reasonable time to
-become involved in the discussion and review process if they wish.
+- Collaborators must be actively contributing to the project
+- A Pull Request must be opened on the @nodejs/nodejs.org README file adding the new collaborator to the list (note the order of names)
+- The Pull Request must be approved by at least two members of @nodejs/website or @nodejs/tsc
+- The Pull Request must remain open for 72 hours without any objections
```
Comment (benhalverson) [inline CONTRIBUTING.md:144]: Which is it 48 or 72 hrs? 
line 119 says 48 line 144 says 72

Comment (ovflowd) [reply CONTRIBUTING.md:144]: If you re-read you will notice that this section (line 139-144) is for Pull Requests for Becoming a Collaborator, whilst line 144 is for regular Pull Requests.

**Review by AugustinMauroy — APPROVED** [review]
LGTM

**Comment by aymen94** [comment]
LGTM 😀

**Review by aymen94 — APPROVED** [review]

**Pull request merged**